packages/gatsby/src/redux/reducers/tasks.js

```diff
@@ -0,0 +1,61 @@
+// TODO: Ensure this is not persisted to disk since tasks are ephemeral
+const initialState = {
+  tasks: new Map(),
```

Comment:
`jobs` is taken 😅 

Will figure out how to migrate the name to that later

Reply:
maybe job2 :-D